.github/workflows/new_issues.yml

```diff
@@ -16,4 +16,5 @@ jobs:
         with:
           project: Openverse
           column: Backlog
+          # TODO: Switch to GITHUB_TOKEN if the project is moved to a repo
```

Comment:
If the project is moved to a certain kind of repo like public/private, maybe? I'm not sure what this comment is trying to say.